src/types/module-checker-result.ts

```diff
@@ -0,0 +1,5 @@
+export default interface IModuleCheckerResult {
```

Comment:
I like to collocate types per modules and exploit typescript duck typings.
When growing, this will become harder to maintain types in my opinion

Reply:
Agreed, let's do this the next time we have to add a new module so it doesn't spiral out of control.